src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -190,4 +196,43 @@ public void setAuthorAliases(Author author, String... aliases) {
             authorAliasMap.put(alias, author);
         }
     }
+
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    public String getLocation() {
+        return location;
+    }
+
+    public String getRepoName() {
+        return repoName;
+    }
+
+    /**
+     * Verifies {@code location} can be presented as a {@code URL} or {@code Path}.
+     * @throws ParseException if otherwise.
+     */
+    private void verifyLocation(String location) throws ParseException {
```

Comment:
Actually, can't we change this to `isPathLocation` and `isUrlLocation` instead, and put them into `CsvParser#processlines`, and make them call different `RepoConfiguration` constructors?
1 which takes a `Path`, and 1 which takes a `URL`?

Reply:
> Actually, can't we change this to isPathLocation and isUrlLocation instead, and put them into CsvParser#processlines, and make them call different RepoConfiguration constructors?
1 which takes a Path, and 1 which takes a URL?

I think this solution is fine, it would be a little redundant to convert String > Path > String again.
However, instead of throwing `ParseException`, perhaps we can make an `InvalidLocationException` instead.